### DISCUSSION TITLE
Fabric crash report

### DISCUSSION THREAD

**Discussion by AI-Computer** [FabricMC/fabric-api#1917] [Player support]
I made a Fabric modpack on the curseforge app but whenever I load it it crashes giving this crash message:

RuntimeException: Non-unique Mixin config name mixins.reach-entity-attributes.json used by the mods feudalw and reach-entity-attributes
<html>at net.fabricmc.loader.impl.launch.FabricMixinBootstrap.init(FabricMixinBootstrap.java:85)<br/>at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:148)<br/>at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:71)<br/>at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)</html>

I did have a mod in the pack called  feudalweapons so I removed that mod but still got the error, and I don't have a mod called  reach-entity-attributes, been trying to fix it for hours but couldn't figure out how,, any help? (I included the download for the mods)
[Fabric pack.zip](https://github.com/FabricMC/fabric/files/7745528/Fabric.pack.zip)

**Comment by warjort** [comment]
Please post the log/latest.log

> RuntimeException: Non-unique Mixin config name mixins.reach-entity-attributes.json used by the mods feudalw and reach-entity-attributes

This says you have two mods trying to share the same configuration file name.
For some reason the feudal weaponary mod you linked does have mixins and configuration for the other mod?

````
$ jar -tf feudal-weaponry-1.0.1.jar | grep reach
mixins.reach-entity-attributes.json
com/knightmere/feudal_weaponry/reachentityattributes/
com/knightmere/feudal_weaponry/reachentityattributes/mixin/
com/knightmere/feudal_weaponry/reachentityattributes/mixin/ServerPlayNetworkHandlerMixin.class
com/knightmere/feudal_weaponry/reachentityattributes/mixin/LivingEntityMixin.class
com/knightmere/feudal_weaponry/reachentityattributes/mixin/client/
com/knightmere/feudal_weaponry/reachentityattributes/mixin/client/GameRendererMixin.class
com/knightmere/feudal_weaponry/reachentityattributes/mixin/PlayerEntityMixin.class
com/knightmere/feudal_weaponry/reachentityattributes/mixin/client/ClientPlayerInteractionManagerMixin.class
com/knightmere/feudal_weaponry/reachentityattributes/mixin/PlayerDropMixin.class
com/knightmere/feudal_weaponry/reachentityattributes/mixin/ServerPlayerInteractionManagerMixin.class
com/knightmere/feudal_weaponry/reachentityattributes/ReachEntityAttributes.class
````

> I did have a mod in the pack called feudalweapons so I removed that mod but still got the error, and I don't have a mod called reach-entity-attributes,

Most likely that is just curseforge being its usual buggy self and it hasn't actually removed what it says it has removed.
The log/latest.log will show what mods are actually being loaded.

If you don't know how to fix it manually, try rebuilding the modpack from scratch without the feudal weaponary mod.

By the way, that feudal weaponary mod is abandoned and the author says it was buggy anyway.
https://www.curseforge.com/minecraft/mc-mods/feudal-weaponry-1-16-5
````
Disclaimer: Comments have been disabled as the mod is no longer being worked on. 

This particular project for Fabric is very bugged
````
